docs/vc-http-api.yml

```diff
@@ -418,7 +494,7 @@ components:
         verifiableCredential:
           $ref: "#/components/schemas/VerifiableCredential"
         options:
-          $ref: "#/components/schemas/LinkedDataProofOptions"
+          $ref: "#/components/schemas/VerifyOptions"
```

Comment:
```suggestion
          $ref: "#/components/schemas/VerifyCredentialOptions"
```

Reply:
This options object is currently shared between the credential verify and presentation verify endpoint